pyEDAA/UCIS/CLI/__init__.py

```diff
@@ -151,18 +153,28 @@ def HandleExport(self, args) -> None:
 		print(f"  IN  -> UCIS (XML):      {ucdbPath}")
 		print(f"  OUT <- Cobertura (XML): {coberturaPath}")
 
-		parser = Parser(ucdbPath)
-		model = parser.get_cobertura_model()
+		parser = Parser(ucdbPath, args.mergeInstances)
+		model = parser.getCoberturaModel()
 
 		with coberturaPath.open('w') as file:
-			file.write(model.get_xml().decode("utf-8"))
+			file.write(model.getXml().decode("utf-8"))
 
 		print()
 
-		coverage = model.lines_covered / model.lines_valid * 100
+		try:
+			lineCoverage = model.linesCovered / model.linesValid * 100
```

Comment:
Is it possible to have no valid line?
If so, isn't the error in counting valid lines instead of the division?

I assume no valid lines would be a broken UCDB file, right?

Reply:
Sorry for late reply I was a little busy.

"valid line" is a term from cobertura coverage.
It's a little misleading and means any code line that can bit hit.
For example comment, empty or excluded line is not count as "valid line".

Reply:
Ah ok. So it's to distinguish code or statements from non-code lines.
Stange naming choice for this property ... anyhow, when Cobertura names it so, it's ok.